### ISSUE TITLE
Save & Spell Pen don't work for custom schools

### ISSUE THREAD

**Issue by mattbford** [drplote/FG-2e-PlayersOption#1]
Setting up these effects for an elementalist caster.

Doesn't seem to work for SAVE: 2 water; or SPELLPEN: 2 water;

**Comment by mattbford** [comment]
I just added them in to data_common_po.lua manually, but perhaps there is a way to do it dynamically.

**Comment by drplote** [comment]
Yeah, looks like you found the workaround. Adding a dynamic option for that wouldn't be bad, but is pretty low on my priority list. Honestly, though, I'm not even sure why it bothers to check that it exists in DataCommon.powertypes first before allowing it to work... damage types have the same issue too (different DataCommon variable, but same basic premise). That's an artifact of how the base 2E ruleset works and my gut feel is I'd rather just rip out that restriction entirely, but I haven't been motivated enough to do it and test it out to see what side effects it has. The "Effects" code is a messy beast.

You probably already know this if you got this far, but for that "water" keyword to work, you'll have to add it to the properties on the cast actions of whatever you're considering water spells.
